apigee_sync.go

```diff
@@ -235,6 +249,14 @@ func downloadDataSnapshot() {
 	scopes = append(scopes, apidInfo.ClusterID)
 	resp := downloadSnapshot(scopes)
 
+	knownTables = extractTablesFromSnapshot(resp)
+
+	db, err := data.DBVersion(resp.SnapshotInfo)
+	if err != nil {
+		log.Panicf("Database inaccessible: %v", err)
```

Comment:
Theoretically, it shouldn't matter, right? I mean, the system can only process one snapshot at s time... so it's naturally single threaded, I believe.